src/binary/ibc-setup/commands/init.spec.ts

```diff
@@ -0,0 +1,89 @@
+import fs from 'fs';
+
+import test from 'ava';
+import axios from 'axios';
+import sinon from 'sinon';
+
+import { Options, run } from './init';
+
+const consoleLog = sinon.stub(console, 'log');
+const fsExistSync = sinon.stub(fs, 'existsSync');
+const fsMkdirSync = sinon.stub(fs, 'mkdirSync');
+const axiosGet = sinon.stub(axios, 'get');
+const fsReadFileSync = sinon.stub(fs, 'readFileSync');
+const fsWriteFileSync = sinon.stub(fs, 'writeFileSync');
+const fsLstatSync = sinon.stub(fs, 'lstatSync');
+
+test.beforeEach(() => {
+  sinon.reset();
+
+  fsLstatSync.returns({
```

Comment:
If you define this as a fake in the outer scope instead of a stub do you need to do anything in the `beforeEach` hook?

Reply:
Yeah, it works nicely with fake: 9b5e421e7cb2bb7618bb5a6beeb27d37eed201ba 🎉